extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.instant.js

```diff
@@ -0,0 +1,56 @@
+require([
+	'wikia.articleVideo.featuredVideo.jwplayer.instance',
+	'wikia.articleVideo.featuredVideo.data',
+	'wikia.articleVideo.featuredVideo.autoplay'
+], function (playerInstance, videoDetails, featuredVideoAutoplay) {
+
+	if (!videoDetails) {
+		return;
+	}
+
+	var videoId = videoDetails.videoId,
+		inNextVideoAutoplayCountries = featuredVideoAutoplay.inNextVideoAutoplayCountries,
+		willAutoplay = featuredVideoAutoplay.willAutoplay;
+
+	function handleTabNotActive(willAutoplay) {
+		var hidden,
+			visibilityChange;
+
+		if (typeof document.hidden !== "undefined") {
+			hidden = "hidden";
+			visibilityChange = "visibilitychange";
+		} else if (typeof document.msHidden !== "undefined") {
```

Comment:
do we need this? according to http://caniuse.com/#search=Page%20Visibility all our browsers support that

Reply:
@hakubo ?